esp/smc/SMCLib/InfoCacheReader.cpp

```diff
@@ -37,7 +49,6 @@ void CInfoCacheReaderThread::threadmain()
             {
                 CCycleTimer timer;
                 Owned<CInfoCache> info = infoCacheReader->read();
-                PROGLOG("CInfoCacheReaderThread %s: InfoCache collected (%u seconds).", name.get(), timer.elapsedMs()/1000);
```

Comment:
I'll need to rebase after HPCC-33895 is merged as this change is actually from that PR.